stripe/src/test/java/com/stripe/android/paymentsheet/ui/BillingAddressViewTest.kt

```diff
@@ -33,4 +36,20 @@ class BillingAddressViewTest {
             billingAddressView.countryView.text.toString()
         ).isEqualTo("United States")
     }
+
+    @Test
+    fun `changing selectedCountry to country without postal code should hide postal code view`() {
+        billingAddressView.selectedCountry = Country("ZW", "Zimbabwe")
+        idleLooper()
+        assertThat(billingAddressView.postalCodeLayout.isVisible)
+            .isFalse()
+    }
+
+    @Test
+    fun `changing selectedCountry to country with postal code should show postal code view`() {
+        billingAddressView.selectedCountry = Country("FR", "France")
```

Comment:
Given that we're still very US-centric, it may be worth adding a test case for the US as well.